### PR TITLE
Mix - Bug fixes

### PR DESCRIPTION
**Pull request by yann300**
- Bugfix: #1397. 
- Debugger panel: UI bug fixes.

### CODE REVIEW

**Comment by gavofyork** [comment]
rebase.

File: mix/ClientModel.cpp [hunk 1]
```diff
@@ -350,6 +350,10 @@ void ClientModel::showDebuggerForTransaction(ExecutionResult const& _t)
 			CompiledContract const* contract = contracts[s.codeIndex];
 			AssemblyItem const& instruction = codeItems[s.codeIndex][instructionIndex];
 
+			std::stringstream str;
+			str << instruction.getLocation().sourceName;
+			qDebug() <<  QString::fromStdString(str.str());
```
Comment (arkpar) [inline mix/ClientModel.cpp]: Do we really need that in debug output?

Comment (yann300) [inline mix/ClientModel.cpp]: I will delete that later, I have to fix one other bug before.

2015-03-26 11:06 GMT+01:00 Arkadiy Paronyan <notifications@github.com>:

Comment (arkpar) [inline mix/ClientModel.cpp]: Does not compile. MixClient::bc should return eth::BlockChain

**Comment by yann300** [comment]
I've pushed a fix for #1386.

**Pull request merged**